src/controllers/action-queue/WyreClient.js

```diff
@@ -83,10 +87,10 @@ export const makeWyreClient = async (opt: WyreClientOptions): Promise<WyreClient
         if (blockchains == null) return
 
         // Special cases for testnets, if they don't yet support (they currently do not)
-        if (blockchains.MUMBAI == null) blockchains.MUMBAI = blockchains.MATIC
+        if (blockchains.MUMBAI == null && blockchains.MATIC != null) blockchains.MUMBAI = blockchains.MATIC
```

Comment:
This doesn't really have any impact?

Reply:
Good catch, this was misguided. The real issue was the test BTC. Still getting used to being allowed to reference nonexistent keys 😂

Reply:
Actually I think still has merit. Don't want to create a MUMBAI key at all if there's no MATIC configured.

Thinking on this a bit more, I don't think there's even a point to setting up MUMBAI at all to a live net address. Plus, MUMBAI really does not behave as reliably as live net

Reply:
Going to just leave this for now. We do have a lot of Object.keys usage throughout the gui for map logic.

Reply:
I think still has merit. Don't want to even create a MUMBAI key at all if there's no MATIC configured.

Thinking on this a bit more, I don't think there's even a point to setting up MUMBAI at all to a live net address. Plus, MUMBAI really does not behave as reliably as live net

Going to just leave this for now. We do have a lot of Object.keys usage throughout the gui for map logic.